setup-functional-tests.sh

```diff
@@ -0,0 +1,11 @@
+git clone --depth 1 https://github.com/mozilla/addons-server.git
+docker-compose -f addons-server/docker-compose.yml -f docker-compose.override.yml up -d --build
```

Comment:
How about renaming `docker-compose.override.yml` to `docker-compose.functional-tests.yml` or something? Also, I'd like to see this moved to a subdirectory within `tests` like `tests/ui` (if that one makes sense).